internal/cli/run/run.go

```diff
@@ -150,6 +150,15 @@ func (c *Command) Run(ctx context.Context) (err error) {
 	// Start the internal app server
 	eg.Go(func() error { return appServer.Run(ctx) })
 	// Wait until either the hot or web server exits
+
+	// Launch browser unless we're told not to
+	if !c.Flag.Noautolaunch { // if not not autolaunch, i.e. if autolaunch
+		go func() {
+			time.Sleep(1 * time.Second)
+			_ = browser.OpenURL("http://" + webln.Addr().String())
```

Comment:
Should check the error

Reply:
this is the only guidance here i don't understand. why? i literally can't think of anything that we'd do differently if it fails or not.  it's an optional convenience feature...

Reply:
My take is that if a user explicitly requested `--open` we should open or fail trying.